project/AppDependencies.scala

```diff
@@ -1,35 +1,35 @@
-import play.core.PlayVersion
+import play.core.PlayVersion.current
 import play.sbt.PlayImport._
 import sbt._
 
 object AppDependencies {
 
   val compile = Seq(
     ws,
-    "uk.gov.hmrc"         %% "bootstrap-play-26"        % "1.13.0",
-    "uk.gov.hmrc"         %% "play-partials"            % "6.11.0-play-26",
-    "uk.gov.hmrc"         %% "url-builder"              % "3.4.0-play-26",
-    "uk.gov.hmrc"         %% "http-caching-client"      % "9.1.0-play-26",
-    "uk.gov.hmrc"         %% "play-language"            % "4.5.0-play-26",
-    "uk.gov.hmrc"         %% "local-template-renderer"  % "2.10.0-play-26",
-    "uk.gov.hmrc"         %% "play-ui"                  % "9.2.0-play-26",
+    "uk.gov.hmrc"         %% "bootstrap-frontend-play-27"        % "5.1.0",
+    "uk.gov.hmrc"         %% "play-partials"            % "8.1.0-play-27",
+    "uk.gov.hmrc"         %% "url-builder"              % "3.5.0-play-27",
+    "uk.gov.hmrc"         %% "http-caching-client"      % "9.4.0-play-27",
+    "uk.gov.hmrc"         %% "play-language"            % "4.13.0-play-27",
+    "uk.gov.hmrc"         %% "local-template-renderer"  % "2.14.0-play-27",
+    "uk.gov.hmrc"         %% "play-ui"                  % "9.0.0-play-27",
     "uk.gov.hmrc"         %% "tax-year"                 % "1.1.0",
-    "uk.gov.hmrc"         %% "domain"                   % "5.10.0-play-26",
-    "uk.gov.hmrc"         %% "auth-client"              % "3.0.0-play-26",
-    "org.reactivemongo"   %% "play2-reactivemongo"      % "0.18.6-play26",
+    "uk.gov.hmrc"         %% "time"                     % "3.19.0",
+    "uk.gov.hmrc"         %% "domain"                   % "5.11.0-play-27",
+    "org.reactivemongo"   %% "play2-reactivemongo"      % "0.18.6-play27",
     "io.lemonlabs"        %% "scala-uri"                % "2.2.3",
     "com.typesafe.play"   %% "play-json-joda"           % "2.6.10"
   )
 
   val test = Seq(
-    "uk.gov.hmrc"             %% "hmrctest"             % "3.9.0-play-26",
-    "org.scalatest"           %% "scalatest"            % "3.0.8",
-    "org.mockito"              % "mockito-all"          % "2.0.2-beta",
-    "org.scalatestplus.play"  %% "scalatestplus-play"   % "3.1.3",
-    "org.pegdown"              % "pegdown"              % "1.6.0",
-    "org.jsoup"                % "jsoup"                % "1.11.3",
-    "com.github.tomakehurst"   % "wiremock-standalone"  % "2.17.0",
-    "com.typesafe.play"       %% "play-test"            % PlayVersion.current
+    "org.scalatest"           %% "scalatest"                % "3.2.8",
```

Comment:
@Christian-QA don't think scalatest is supposed to run with scalatestplus-play > v3.0.x we should downgrade this back to 3.0.8

Reply:
Done. Had to redo all the must matchers imports, so the change will look deceptively big

Reply:
I originally needed to upgrade it to 3.2.x because it was throwing
`java.lang.NoSuchMethodError: org.scalactic.BooleanMacro.<init>(Lscala/reflect/macros/whitebox/Context;Ljava/lang/String;)V`

The error doesn't specify a spec, and with the upgrade resolving the issue, I never discovered what caused it. Will give it another look